src/FluentValidation.Tests/ExactLengthValidatorTester.cs

```diff
@@ -37,6 +37,14 @@ public class ExactLengthValidatorTester {
 			result.IsValid.ShouldBeTrue();
 		}
 
+		[Fact]
+		public void When_the_text_contains_grapheme_clusters_then_the_validator_should_count_each_exactly_once()
+		{

```

Comment:
Please could move the opening brace be on the end of the preceding line ([as per our coding standard](https://github.com/FluentValidation/FluentValidation/blob/main/.github/CONTRIBUTING.md#contributing-code-and-content))? Thanks!